addons/binding/org.openhab.binding.robonect/README.md

```diff
@@ -0,0 +1,123 @@
+| `job#afterMode`          | String                    | Sets the mode the mower should be put in after a mowing Job. (See JOB on channel `owerStatus#mode`)
+| `job#start`              | String                    | Sets the start time for the next job in the form hh:mm  |
+| `job#end`                | String                    | Sets the end time for the next job in the form hh:mm |
+| `timer#status`           | String                    | Retrieves the status of the timer which can be <ul><li>INACTIVE : no timer set</li><li>ACTIVE - timer set and currently running</li><li>STANDBY - timer set but not triggered/running yet</li></ul> |
+| `timer#nextTimer`        | DateTime                  | Retrieves the Date and Time of the next timer set. This is just valid if there is an ACTIVE timer status (see `timer#status`). |
+| `wlan#signal`            | Number                    | Retrieves the current WLAN Signal strength in dB |
+| `error#code`             | Number                    | The mower manufacturer code in case the mower is in status 7 (error). The binding resets this to UNDEF, once the mower is not in error status anymore. |
+| `error#message`          | String                    | The error message in case the mower is in status 7 (error). The binding resets this to UNDEF, once the mower is not in error status anymore. |
+| `error#date   `          | DateTime                  | The date and time the error happened. The binding resets this to UNDEF, once the mower is not in error status anymore. |
+| `lastError#code`         | Number                    | The mower manufacturer code of the last error happened |
+| `lastError#message`      | String                    | The error message of the last error happened |
+| `lastError#date   `      | DateTime                  | The date and time of the last error happened |
+| `health#temperature`     | Number                    | The temperature of the mower (just available for robonect firmware >= 1.0) |
+| `health#humidity   `     | Number                    | The humidity of the mower (just available for robonect firmware >= 1.0) |
+| `version#serial`         | String                    | Retrieves the serial number of the Robonect Module |
+| `version#version`        | String                    | Retrieves the firmware version of the Robonect Module |
+| `version#compiled`       | String                    | Retrieves the compile date and time of the Robonect firmware version |
+| `version#comment`        | String                    | Retrieves the firmware version comment |
+    
+
+## Full Example
+
+Things file `.things`
+
+```
+Thing robonect:mower:automower "Mower" @ "Garden" [ host="192.168.2.1", pollInterval=5, user="gardener", password = "cutter"]
+```
+
+Items file `.items`
+
+```
+String mowerName "Mower name" {robonect:mower:automower:mowerInfo#name"}
```

Comment:
Maybe you could format the items definitions a bit to improve the readability.

Reply:
good idea. done.